src/main/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtil.java

```diff
@@ -359,7 +359,7 @@ public static NewUserResponse findUserProfileStatusByEmail(String emailAddress,
                 if (errorResponse != null) {
                     log.error("{}:: Response from UserProfileByEmail service call {}",
                         loggingComponentName, errorResponse.getErrorDescription());
-                }                
+                }

```

Comment:
perhaps undo changes in this file as its just white space updates and not needed

Reply:
The base branch has this - ill remove from there